src/build.sh

```diff
@@ -111,7 +131,46 @@ cmake --install build
 popd
 
 pushd v2i-hub
-cmake -Bbuild -DCMAKE_PREFIX_PATH=\"/usr/local/share/tmx\;/opt/carma/cmake\;\" -DqserverPedestrian_DIR=/usr/local/share/qserverPedestrian/cmake -Dv2xhubWebAPI_DIR=/usr/local/share/v2xhubWebAPI/cmake/ -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS}" -DCMAKE_C_FLAGS="${CMAKE_CXX_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" -DSAEJ2735_SPEC_VERSION="${J2735_VERSION}" .
+
+# Dynamically list plugin directories (those with CMakeLists.txt, excluding main dir)
+plugin_dirs=()
+for f in */CMakeLists.txt; do
+  dir=$(dirname "$f")
+  if [ "$dir" != "." ] && [ "$dir" != "build" ]; then
+    plugin_dirs+=("$dir")
+  fi
+done
+
+# Plugin selection: Use --plugins arg if provided, else prompt
+if [[ -z "$PLUGIN_INPUT" ]]; then
```

Comment:
Why are we getting this input again here. The script already fails if its not provided as an argument